paddle/fluid/operators/dropout_op.cu

```diff
@@ -47,7 +47,11 @@ __global__ void RandomGenerator(const size_t n, const int seed,
     if (dist(rng) < dropout_prob) {
       mask = static_cast<T>(0);
     } else {
-      mask = static_cast<T>(1);
+      if (div_prob_in_train) {
+        mask = static_cast<T>(1.0f / (1.0f - dropout_prob));
```

Comment:
`div` is time consuming, so, you had better precompute `static_cast<T>(1.0f / (1.0f - dropout_prob))` outside of `for`.